### PR TITLE
Adding Release Dockerfile

### PR DESCRIPTION
**Pull request by muditatandon**
Adding release folder which will contains the Dockefile used to create the docker image used by Louhi workflows for 3p flutter releases.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: release/DOCKERFILE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu
```
Comment (godofredoc) [inline release/DOCKERFILE:1]: Should we call this file Dockerfile to make it consistent with the names in other folders?

**Review by keyonghan — COMMENTED** [review]
1. Can we add a README file to start documenting what this release folder does?
2. Please update the PR description to reflect what this PR does.

File: release/DOCKERFILE (hunk 1 above)
Comment (godofredoc) [reply release/DOCKERFILE:1]: Let's also pin the dependency to `FROM ubuntu:bionic@sha256:eb1392bbdde63147bc2b4ff1a4053dcfe6d15e4dfd3cce29e9b9f52a4f88bc74`

**Review by godofredoc — COMMENTED** [review]
Let's also add a dependabot entry here: https://github.com/flutter/cocoon/blob/main/.github/dependabot.yml#L71 for this folder.

File: release/DOCKERFILE (hunk 1 above)
Comment (muditatandon) [reply release/DOCKERFILE]: Updated

**Review by godofredoc — APPROVED** [review]
Thanks for the changes!

**Comment by godofredoc** [comment]
\cc @itsjustkevin FYI

File: release/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Flutter Release Docker Image
```
Comment (godofredoc) [inline release/README.md:1]: Somewhere in this file there is hidden character making the analysis fail. I'd recommend running `dart analyze` locally to find out the character and remove it.

Comment (muditatandon) [reply release/README.md:1]: I ran it locally, found no issues - 

`muditatandon-macbookpro% dart analyze README.md
Analyzing README.md...                 0.7s
No issues found!
muditatandon-macbookpro% 
muditatandon-macbookpro% dart analyze
Analyzing release...                   1.1s
No issues found!`

File: release/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+The release folder contains the Dockerfile which is used to create the docker image used by Louhi workflows for 3P releases.
```
Comment (drewroengoogle) [inline release/README.md:3]: I think this is why the build is failing, you've got a trailing space on this line:
```suggestion
The release folder contains the Dockerfile which is used to create the docker image used by Louhi workflows for 3P releases.
```

Comment (muditatandon) [reply release/README.md:3]: Oh let me remove it, the `dart analyze` didn't show it.

Comment (muditatandon) [reply release/README.md]: That solved the analyze step, it failing at the app_dart step. 

Looking into that.

File: release/README.md (hunk 2 above)
Comment (ricardoamador) [reply release/README.md:1]: Hi Mudita, try updating the mocks in both app_dart and auto_submit:
```
flutter pub run build_runner build
```
Run the above command in both of those directories then commit the changes and push.

**Pull request merged**